addons/portal/views/portal_templates.xml

```diff
@@ -488,6 +488,21 @@
             <label class="col-form-label" for="city">City</label>
             <input type="text" name="city" t-attf-class="form-control #{error.get('city') and 'is-invalid' or ''}" t-att-value="city or partner.city" />
         </div>
+        <div t-attf-class="mb-3 #{error.get('street2') and 'o_has_error' or ''} col-xl-6">
+            <label class="col-form-label" for="street2">Street 2</label>
+            <input type="text" name="street2" t-attf-class="form-control #{error.get('street2') and 'is-invalid' or ''}" t-att-value="street2 or partner.street2"/>
+        </div>
+        <div t-attf-class="mb-3 #{error.get('state_id') and 'o_has_error' or ''} col-xl-6">
+            <label class="col-form-label label-optional" for="state_id">State / Province</label>
+            <select name="state_id" t-attf-class="form-select #{error.get('state_id') and 'is-invalid' or ''}">
+                <option value="">select...</option>
+                <t t-foreach="states or []" t-as="state">
+                    <option t-att-value="state.id" style="display:none;" t-att-data-country_id="state.country_id.id" t-att-selected="state.id == int(state_id) if state_id else state.id == partner.state_id.id">
```

Comment:
Hello @khpa-odoo :smile: 
As we're updating those lines anyway, we could probably use the opportunity to change the ugly `style="display:none;"` to a `d-none` class ? :slightly_smiling_face:
Thanks!

Reply:
Hello @amdi-odoo :blush: 
Sure! Updated the code :+1:

Reply:
`d-none` isn't dynamic (no use of variables) so simply a `class` instead of the `t-att-class` should do it :smile: 
Thanks!

Reply:
Done : )